src/client/components/SignInButton/Button.js

```diff
@@ -5,7 +5,10 @@ import PropTypes from 'prop-types';
 const Button = (props) => {
   const { onClick } = props;
   const clicked = () => {
+    const clienId = process.env.SLACK__CLIENT_ID;
     onClick();
+    const oauthUrl = `https://slack.com/oauth/v2/authorize?client_id=${clienId}&user_scope=identity.basic`;
+    window.location = oauthUrl;
```

Comment:
This whole code should go to `LoginPage.component.js` in the `onClick` handler for the `Button` there.